src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -17,6 +17,17 @@
 
 class PrettyPageHandler extends Handler
 {
+    const EDITOR_SUBLIME = "sublime";
```

Comment:
maybe we can add an inline comment with a link to https://github.com/filp/whoops/blob/master/docs/Open%20Files%20In%20An%20Editor.md ?

Reply:
I would expect these constants to appear somewhere else. And there is a place to put them: `$this->editors` is the array defining return strings for 9 of these 10 constants.

Why are there 10 constants, and why is the `setEditor` method throwing an exception on anything that isn't either defined inside that array or otherwise a callable? Seems like one of these constants will probably not work right now.

Reply:
Answer: xdebug is added dynamically based on the extension being installed or not. So xdebug isn't reliably available and will throw exceptions on systems where the extension is missing if you run `setEditor('xdebug')` (or use the new constant).